src/Forms/__tests__/ValidationErrorMessage.test.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { render, cleanup } from 'test-utils';
+import ValidationErrorMessage from '../ValidationErrorMessage';
+
+describe('Validation Error Message', function () {
+  let getByText;
+  beforeEach(function () {
+    ({ getByText } = render(
+      <ValidationErrorMessage>
+        Error: This entry may only contain numbers.
+      </ValidationErrorMessage>
+    ));
+  });
+  afterEach(function () {
+    cleanup();
+  });
```

Comment:
This can be dropped (I'll submit a fix today for seas-computing/course-planner#149 today)

Reply:
Ok, sounds good. Fixed through 45f02c7